integration-tests/environments/react-native-test-app/ios/Podfile

```
@@ -2,6 +2,8 @@ require_relative '../../../../node_modules/react-native-test-app/test_app'
 
 workspace 'RealmTests.xcworkspace'
 
+ENV["USE_CCACHE"] = ENV["USE_CCACHE"] || Pod::Executable::which('ccache') ? "1" : "0"
```

Think this got reverted 😛 
```suggestion
ENV['USE_CCACHE'] = ENV['USE_CCACHE'] || Pod::Executable::which('ccache') ? '1' : '0'
```